data/templates/css/wikimedia.scss

```diff
@@ -1,5 +1,4 @@
 @import "project-setup";
-@import "compass";
```

Comment:
Done with the diff, very few cases for image-height/width, therefore I agree with @mattdangerw here lets just replace it with actual values.